modules/artifact-activation/src/test/resources/apps/basic-with-supported-java-versions/src/main/java/org/test/TestClass.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2023 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.test;
+
+public class TestClass {
+
+}
```

Comment:
Shouldn't we simplify the test artifact with just the basics needed for the test case?
It looks like just an empty Mule app with the required mule-artifact.json and the most basic pom.xml with no dependencies should be enough.
We could remove the test-script.dwl, test-resource.yaml and TestClass.java files as well as trimming the pom.xml a little.